Validation/MuonHits/src/CSCSimHitMatcher.cc

```diff
@@ -237,7 +242,7 @@ void CSCSimHitMatcher::fitHitsInChamber(unsigned int detid, float& intercept, fl
 
   std::unique_ptr<TGraphErrors> gr(new TGraphErrors(x.size(), &x[0], &y[0], &xe[0], &ye[0]));
   std::unique_ptr<TF1> fit(new TF1("fit", "pol1", -3, 4));
-  gr->Fit("fit", "EMQ");
+  gr->Fit("fit", "EMQN");
```

Comment:
This version of `Fit` should also be changed. You should pass in the `TF1` you created as the first argument instead of using the string name.

Reply:
@dildick  Please address above comment before launching the test.

Reply:
Done.